scripts/circleci/release/create-agent-installer.sh

```diff
@@ -358,12 +358,50 @@ create_alt_yum_repo_packages;
 
 tar -cf repo_packages.tar *bootstrap*.rpm *bootstrap*.deb
 
+set -e
+
+REPOSITORY_URL="https://scalyr-repo.s3.amazonaws.com/$REPO_BASE_URL"
+
+PUBLIC_KEY_URL="https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x84AC559B5FB5463885CE0841F70CEEDB4AD7B6C6"
+
+# create a yum repo spec data.
+YUM_REPO_SPEC=$(cat << EOM
+[scalyr]
+includepkgs=scalyr-agent,scalyr-agent-2,scalyr-repo
+name=Scalyr packages - noarch
+baseurl=https://scalyr-repo.s3.amazonaws.com/$RELEASE_REPO_BASE_URL/yum/binaries/noarch
+mirror_expire=300
+metadata_expire=300
+enabled=1
+gpgcheck=1
+gpgkey=${PUBLIC_KEY_URL}
```

Comment:
One option would also be to host that key file on our website / s3 bucket, wdyt?

Reply:
But using ubuntu keyserver should also work for now.